ee/src/main/java/org/jboss/as/ee/subsystem/ContextServiceRemove.java

```diff
@@ -35,9 +33,4 @@
     private ContextServiceRemove() {
         super(ContextServiceAdd.INSTANCE);
     }
-
-    @Override
-    protected ServiceName serviceName(String name) {
-        return ConcurrentServiceNames.getContextServiceServiceName(name);
-    }
```

Comment:
Given that you are no longer overriding behavior of ServiceRemoveStepHandler, we can eliminate this class altogether.

Reply:
it's needed to remove the service created by the add handler

Reply:
ok never mind, I got what you mean from other comment

Reply:
done